pkg/webui/console/views/device-overview/index.js

```diff
@@ -223,6 +251,29 @@ class DeviceOverview extends React.Component {
 
     sheetData.push(sessionInfoData)
 
+    const macStateAndSettings = {
+      header: 'MAC data',
+      items: [],
+    }
+
+    if (Boolean(mac_state) && Boolean(mac_settings)) {
+      macStateAndSettings.items.push({
+        value: (
+          <ModalButton
+            modalData={{
+              message: session ? m.hasSession : m.noSession,
+            }}
+            onApprove={this.onExport}
+            message={m.macData}
+            type="button"
+            icon="file_download"
+          />
+        ),
+      })
+    }
```

Comment:
This condition doesn't make sense to me. The button should always be visible, the only thing that can change is whether the `mac_state` is present, but that doesn't affect button visibility.

```suggestion
    const macStateAndSettings = {
      header: 'MAC data',
      items: [{
        value: (
          <ModalButton
            modalData={{
              message: session ? m.hasSession : m.noSession,
            }}
            onApprove={this.onExport}
            message={m.macData}
            type="button"
            icon="file_download"
          />
        ),
      }],
    }
```